ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__ibmcloud-4.18-nightly-x86.yaml

```diff
@@ -53,6 +53,40 @@ tests:
     - chain: openshift-qe-ingress-perf
     workflow: openshift-qe-installer-ibmcloud-ipi-ovn
   timeout: 5h0m0s
+- always_run: false
+  as: node-density-heavy-etcd-fips-24nodes
+  steps:
+    allow_skip_on_success: true
+    cluster_profile: ibmcloud-qe
+    env:
+      COMPUTE_NODE_REPLICAS: "24"
```

Comment:
for some reason, this number isn't actually setting 24 worker nodes 

You can use these similar steps to first create a 3 node cluster and then scale up to 24 worker nodes like: 
https://github.com/openshift/release/blob/5013e84941df6fba3b25567ea7e502a67f1693cc/ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__ibmcloud-4.17-nightly-x86.yaml#L95-L98

Reply:
Done, thanks!